### PR TITLE
Update to use Java 11

### PR DESCRIPTION
**Pull request by tshoang**
Update to Java 11

### CODE REVIEW

File: ch.ethz.eventb.utils-feature/feature.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
 <feature
```
Comment (cfsnook) [inline ch.ethz.eventb.utils-feature/feature.xml:16]: should update year to 2021 in copyright header

File: ch.ethz.eventb.utils.sdk/feature.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
 <feature
```
Comment (cfsnook) [inline ch.ethz.eventb.utils.sdk/feature.xml:15]: should update year to 2021 in copyright header

**Review by cfsnook — CHANGES_REQUESTED** [review]
see detail comments about copyright headers in feature.xml files

File: ch.ethz.eventb.utils.sdk/feature.xml (hunk 2 above)
Comment (tshoang) [reply ch.ethz.eventb.utils.sdk/feature.xml:15]: Fixed in commit ab872f1e6fd24630f61aeda36aa29f1c2d693185.

File: ch.ethz.eventb.utils-feature/feature.xml (hunk 1 above)
Comment (tshoang) [reply ch.ethz.eventb.utils-feature/feature.xml:16]: Fix in commit ab872f1e6fd24630f61aeda36aa29f1c2d693185.

**Review by cfsnook — APPROVED** [review]
much better.. and a few other useful changes sneaked in?

**Comment by tshoang** [comment]
Yes, some useful things from GitHub, including CodeQL. There is one issue left from CodeQL. Unfortunately, the Rodin Update Site is only available via HTTP.

**Pull request merged**